package.json

```diff
@@ -212,7 +212,7 @@
             {
               "name": "Properties",
               "gridItemConfig": {
-                "sizex": 1,
+                "sizex": 2,
```

Comment:
Added slightly longer text for IOPS and Throughput for Next-Gen GP so I had to increase this as well: